dev/build/windows/makecoq_mingw.sh

```diff
@@ -63,6 +63,12 @@ RMDIR_BEFORE_BUILD=1
 #   Relocatble builds are first configured with prefix=./ then build and then
 #   reconfigured with prefix=<installroot> before make install.
 
+###################### COMPONENT VERSIOn OVERRIDES #####################
+
+# This is a temporary solution until the overlay mechanism is finalized
+
+# VST master did not work on 2018-08-19 (365d1b2d42eb31fff58bac560c078858602ba436)
+VST_CI_BRANCH=e49605cf1f1e5ae3bbec3d6554122427a94ae986
```

Comment:
VST master has been fixed in the meantime.

Reply:
Thanks for the hint on VST - I will change it!

I have one strange effect: at home I get bignums-649dad440287151fd1937defb852886f192b8820 which is the head of v8.8 branch, not from master. On CI I get the head of the master branch which seems to fail. Any idea what is going on here?

Reply:
That's weird, could it be a caching problem? In any case, the diff between the tip of `v8.8` and `master` is limited to `.gitignore`, `.travis.yml` and `default.nix` (https://github.com/coq/bignums/compare/v8.8...master?expand=1) not something that should affect the build.

Reply:
The v8.8 vs master issue with bignums is my bad: I edit the build files in a branch which is based on master, but when I build it I didn't use the local option, but build Coq 8.8.1, which has a difference ci_basic_overlays file which uses bignums v8.8. Time to get rid of this "build something else than you have" mess.

Still need to find out why CI fails ...

Reply:
@Zimmi48 : VST master (https://github.com/PrincetonUniversity/VST/commit/4ad211edd35cd21b3ffe1a63871cfe2ed914d183) definitely does not build, at least not in default configuration or if Compcert is not installed. Right now I cannot test it with compcert beeing installed because compcert requires menhirlib and menhirlib seems to have a regression - not sure how compcert is built in CI. I attached the VST error log.
The pinned version (e49605cf1f1e5ae3bbec3d6554122427a94ae986) does compile fine. So I keep the pinned version. I will discuss this with Andrew and Lennart.

Reply:
Sorry, forgot the log ...

[VST-4ad211edd35cd21b3ffe1a63871cfe2ed914d183-make.log.txt](https://github.com/coq/coq/files/2321652/VST-4ad211edd35cd21b3ffe1a63871cfe2ed914d183-make.log.txt)
[VST-4ad211edd35cd21b3ffe1a63871cfe2ed914d183-make.err.txt](https://github.com/coq/coq/files/2321653/VST-4ad211edd35cd21b3ffe1a63871cfe2ed914d183-make.err.txt)